dev/ColorPicker/ColorPicker.cpp

```diff
@@ -301,6 +301,10 @@ void ColorPicker::OnPropertyChanged(winrt::DependencyPropertyChangedEventArgs co
     {
         OnColorSpectrumComponentsChanged(args);
     }
+    else if (property == s_OrientationProperty)
+    {
+        OnUpdateOrientation(args);
```

Comment:
The naming convention is On*Changed, so I would change this to `OnOrientationChanged`